infinitest-eclipse/src/main/java/org/infinitest/eclipse/prefs/SwtColorFieldEditor.java

```diff
@@ -0,0 +1,33 @@
+package org.infinitest.eclipse.prefs;
+
+import org.eclipse.jface.preference.*;
+import org.eclipse.swt.*;
+import org.eclipse.swt.widgets.*;
+
+public class SwtColorFieldEditor extends ComboFieldEditor {
```

Comment:
Wasn't it possible to use the builtin org.eclipse.jface.preference.ColorFieldEditor?